atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -311,6 +312,33 @@ public void deleteRange(TableReference tableRef, RangeRequest range) {
         }
     }
 
+    @Override
+    public void deleteAllTimestamps(TableReference tableRef,
+            Map<Cell, Long> maxTimestampExclusiveByCell) {
+        deleteAllTimestampsDefaultImpl(this, tableRef, maxTimestampExclusiveByCell);
+    }
+
+    public static void deleteAllTimestampsDefaultImpl(KeyValueService kvs, TableReference tableRef,
```

Comment:
We should remember to inline this once JDBC can be removed. Incidentally why couldn't JDBC just use the abstract KVS's implementation?

Reply:
it doesn't extend AbstractKvs - I tried to make it, but it involved some changes, which I didn't think were worth it given this is deprecated